docs/general/descriptions.md

```diff
@@ -0,0 +1,127 @@
+---
+title: Descriptions
+---
+
+# Descriptions
+
+Documentation is first‐class feature of GraphQL, and human-readable
+descriptions can be added into all GraphQL types, fields and
+other definitions.
+
+These are often shown by interactive GraphQL clients (Like
+[GraphiQL](https://github.com/graphql/graphiql) and documentation generators.
+
+The preferred way of adding description in strawberry is using the
+`description=` argument in strawberry decorators:
+
+```python+schema
+    @strawberry.enum(description="Example enum")
+    class EnumType(Enum):
+        FOO = "foo"
+        BAR = "bar"
+
+    @strawberry.type(description="The main GraphQL type")
+    class Query:
+        enum: EnumType = strawberry.field(default=EnumType.BAR, description="A dataclass field")
+
+        @strawberry.field(description="A GraphQL field with a resolver and arguments")
+        def resolver(self, arg1: str, arg2: int) -> int:
+            return 1
+
+    schema = strawberry.Schema(query=Query)
```

Comment:
we don't need these spaces at the beginning 😊

Reply:
Sorry :sweat_smile: